locales/en/app.ftl

```diff
@@ -514,6 +514,13 @@ want-to-add = Want to add another email?
 # the form to add an additional email to Firefox Monitor.
 verify-the-link = Verify the link sent to { $userEmail } to add it to {-product-name}.
 
+## These are part of a confirmation page that appears after a user has verified
+## an additional email to Firefox Monitor.
+
+email-verified = Email Successfully Verified!
+email-added-to-subscription = We’ll alert you if { $email } appears in a data breach.
+email-verified-view-dashboard = To see and manage all emails you’ve signed up for breach monitoring, { $sign-in-link }.
+
```

Comment:
Still missing the `##` to reset the section comment (otherwise it applies until the end of the file, or the next section comment)

Reply:
updated.